### ISSUE TITLE
How to compile .phar

### ISSUE THREAD

**Issue by HittmanA** [ClearSkyTeam/ClearSky#522]
Since your Jenkins is broken how do I compile a .phar my self?

**Comment by Aericio** [comment]
Just use the "scr" file, you don't need to compile a phar. If you do want do though, download DevTools and put the src file in your minecraft server folder, **not plugins**. Make sure its only the "src" and not another .phar inside the server folder. Run it and type in "makeserver" and there you go. It'll take up to 5 minutes, though.

**Comment by HittmanA** [comment]
Ok thank you so much!

**Comment by Aericio** [comment]
Close please :D

**Comment by HittmanA** [comment]
Wait which folder do i put src in?

**Comment by Aericio** [comment]
In your MCPE server folder, where the start.bat or run.cmd or whatever is

**Comment by Aericio** [comment]
![image](https://cloud.githubusercontent.com/assets/16523741/16174987/3aa6c61c-3579-11e6-937e-386d51c5bb16.png)
Put it in here

**Comment by HittmanA** [comment]
Ok thank you again!

**Comment by HittmanA** [comment]
It keeps getting to 
[22:26:56] [Server thread/INFO]: [DevTools] Adding src/spl/UndefinedPropertyException.php 
and then it stops. Is this normal?

**Comment by Aericio** [comment]
Its normal, thats what it is supposed to do.

**Comment by HittmanA** [comment]
Ok Thanks again 😆

**Comment by HittmanA** [comment]
No it doesn't resume, but i forgot to remove the other clear sky .phar so let me try again.

**Comment by HittmanA** [comment]
Hmm you know what I'm having trouble doing this. Could you just give me the .phar that you have?

**Comment by Aericio** [comment]
Sure.

**Comment by HittmanA** [comment]
Thank you so much!

**Comment by Aericio** [comment]
[ClearSky.zip](https://github.com/ClearSkyTeam/ClearSky/files/322085/ClearSky.zip)
Latest version, as of now at least

**Comment by HittmanA** [comment]
Thank you so so much! i will put a sign to thank you on my server!

**Comment by HittmanA** [comment]
The .phar does not work. I put it into my server folder and started my server. It is listed as offline and is unjoinable. It also says it is build #393 or something. Why is this?

**Comment by Aericio** [comment]
It is supposed to be #CuttingEdge... Make sure its the correct phar.

**Comment by HittmanA** [comment]
Ok thanks. So just upload the src file to my server folder and thats it?

**Comment by robske110** [comment]
yes and remove any phar file
use php7-0.15

**Comment by HittmanA** [comment]
Ok thank you so much guys!

**Comment by Aericio** [comment]
Download .zip, then open it and take out the src file and put it in your server 
[php7-0.15 GitHub](https://github.com/ClearSkyTeam/ClearSky/tree/php7-0.15)

**Comment by HittmanA** [comment]
Says i am running build #343 and won't show as online or let me join.

**Comment by robske110** [comment]
Remove all phars

**Comment by Aericio** [comment]
Remove other ClearSky.phar or Pocketmine-MP.phar, just have src

**Comment by HittmanA** [comment]
I did

**Comment by robske110** [comment]
Locking conversation for help please join the CS group (@Aerico whereare you?)
